lib/orbital/api.rb

```diff
@@ -2,6 +2,8 @@ module Killbill #:nodoc:
   module Orbital #:nodoc:
     class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
+      FIVE_MINUTES_AGO = (1 * 300)
```

Comment:
Well I didn't see an easy way to make it configurable (such as consuming a property that is passed in); but I made a simple enough modification in 95b4341 to return `CANCELED` status for `UNDEFINED`.